build/build-release.sh

```diff
@@ -46,7 +46,7 @@ build_release() {
     GOOS=linux GOARCH=amd64 make build
     GOOS=windows GOARCH=amd64 make build
     mv opa_windows_amd64 opa_windows_amd64.exe
-    mv opa_*_* $OUTPUT_DIR
+    cp opa_*_* $OUTPUT_DIR
```

Comment:
If we switch to this it means older releases that are missing `build/gen-deb.sh` will leave behind binaries.

If you move the call to `build_distro_packages` to be in this function right after we build the targets it should simplify some of the error cases/cleanup for that.

Reply:
For sure, didn't know if it was ok to have everything in one function. I thought that the binary that would be left behind is on the docker image which is ephemeral is that not the case?

Reply:
Please see updated changes. Moved the call to `build_distro_packages` into this function as requested.